cmd/apply.go

```diff
@@ -1,9 +1,11 @@
 package cmd
 
 import (
-	"github.com/speakeasy-api/openapi-overlay/pkg/loader"
-	"github.com/spf13/cobra"
 	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/vnarek/openapi-overlay/pkg/loader"
```

Comment:
Is this from your fork? 

Should stay a module of this project: `github.com/speakeasy-api/openapi-overlay/pkg/loader`